ConcernsCaseWork/ConcernsCaseWork.Shared.Tests/Factory/NTIUnderConsiderationFactory.cs

```diff
@@ -1,32 +1,50 @@
-﻿using AutoFixture;
-using ConcernsCaseWork.Enums;
-using ConcernsCaseWork.Models.CaseActions;
-using System;
-using System.Collections.Generic;
-
-namespace ConcernsCaseWork.Shared.Tests.Factory
-{
-	public static class NTIUnderConsiderationFactory
-	{
-		private readonly static Fixture Fixture = new Fixture();
-
-		public static List<NtiUnderConsiderationModel> BuildListNTIUnderConsiderationModel(DateTime? closedAt = null)
-		{
-			return new List<NtiUnderConsiderationModel>
-			{
-				BuildNTIUnderConsiderationModel(closedAt)
-			};
-		}
-
-		public static NtiUnderConsiderationModel BuildNTIUnderConsiderationModel(DateTime? closedAt = null)
+﻿using AutoFixture;
+using ConcernsCaseWork.Models.CaseActions;
+using System;
+using System.Collections.Generic;
+
+namespace ConcernsCaseWork.Shared.Tests.Factory
+{
+	public static class NTIUnderConsiderationFactory
+	{
+		private readonly static Fixture Fixture = new Fixture();
+
+		public static List<NtiUnderConsiderationModel> BuildListNTIUnderConsiderationModel()
+		{
+			return new List<NtiUnderConsiderationModel>
+			{
+				BuildNTIUnderConsiderationModel(),
+				BuildClosedNTIUnderConsiderationModel()
+			};
+		}
+		
+		public static List<NtiUnderConsiderationModel> BuildClosedListNTIUnderConsiderationModel()
+		{
+			return new List<NtiUnderConsiderationModel>
+			{
+				BuildClosedNTIUnderConsiderationModel(),
+				BuildClosedNTIUnderConsiderationModel()
+			};
+		}
+
+		public static NtiUnderConsiderationModel BuildNTIUnderConsiderationModel()
 		{
 			var ntiUnderConsideration = new NtiUnderConsiderationModel();
 			ntiUnderConsideration.Id = Fixture.Create<long>();
 			ntiUnderConsideration.CaseUrn = Fixture.Create<long>();
 			ntiUnderConsideration.Notes = Fixture.Create<string>();
-			ntiUnderConsideration.ClosedAt = closedAt;
 
 			return ntiUnderConsideration;
-		}
-	}
+		}
+		public static NtiUnderConsiderationModel BuildClosedNTIUnderConsiderationModel()
+		{
+			var ntiUnderConsideration = BuildNTIUnderConsiderationModel();
+			

```

Comment:
You should be able to use AutoFixture to create the entire ntiUnderConsideration object, unless there's something that stops it. Just an observation.